app/main/views/services.py

```diff
@@ -187,57 +187,56 @@ def start_new_draft_service(framework_slug, lot_slug):
     )
 
     section = content.get_section(content.get_next_editable_section_id())
+    if section is None:
+        section = content.get_section(content.get_next_section_id(None))
+        while (section and not section.edit_questions):
+            section = content.get_section(content.get_next_section_id(section.id))
+
+        if section is None:
+            abort(404)
+
+        section = section.get_question_as_section(section.get_next_question_slug())
+
+    if request.method == 'POST':
+        update_data = section.get_data(request.form)
+
+        try:
+            draft_service = data_api_client.create_new_draft_service(
+                framework_slug, lot['slug'], current_user.supplier_id, update_data,
+                current_user.email_address, page_questions=section.get_field_names()
+            )['services']
+        except HTTPError as e:
+            update_data = section.unformat_data(update_data)
+            errors = section.get_error_messages(e.message)
+
+            return render_template(
+                "services/edit_submission_section.html",
+                framework=framework,
+                section=section,
+                service_data=update_data,
+                errors=errors
+            ), 400
+
+        return redirect(
+            url_for(
+                ".view_service_submission",
+                framework_slug=framework['slug'],
+                lot_slug=draft_service['lotSlug'],
+                service_id=draft_service['id'],
+            )
+        )
 
     return render_template(
         "services/edit_submission_section.html",
         framework=framework,
+        lot=lot,
+        one_service_limit=lot['oneServiceLimit'],
```

Comment:
If the `lot` object is being passed to the template then I don't know if we also need the `one_service_limit` as a separate parameter - it's probably fine for the template to do `lot.one_service_limit`.